mariadb_copy.md

```diff
@@ -30,9 +30,10 @@ just illustrative, use values which are correct for your environment:
 
 ```
 EXTERNAL_MARIADB_IP=192.168.24.3
+EXTERNAL_DB_ROOT_PASSWORD=SomePassword
```

Comment:
Maybe something like

`EXTERNAL_DB_ROOT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep 'MysqlRootPassword' | tr -d ' ' | cut -d ':' -f 2)`

Is a bit more illustrative than SomePassword or 12345678 at least that will show a place where the passwords might be located.

Reply:
Thanks that's a good point, updating it.